SimpleRss/SimpleRss/ViewControllers/FeedViewController.swift

```diff
@@ -67,7 +68,7 @@ extension FeedViewController: UITableViewDataSource {
         let feed = feedList[indexPath.row]
         
         cell.titleLabel.text = feed.title
-        cell.previewImageView.downloaded(from: feed.picUrl)
+        cell.imageUrl = URL(string: feed.picUrl ?? "")
```

Comment:
As I can see imageUrl can accept Optional value. Don't use nil-coalescing here.

Reply:
but url doesn't accept optional value

Reply:
Unwrap optional using `if let` for instance